src/plonk.rs

```diff
@@ -155,6 +150,10 @@ pub enum Error {
     OpeningError,
     /// Transcript error
     TranscriptError,
+    /// Insufficient number of rows available
```

Comment:
```suggestion
    /// Instance provided has fewer than the minimum rows required.
```

Reply:
Hm, while the comment as-is is confusing I don't quite think this is the correct fix. I will change to

"Instance provided has more rows than supported by circuit"